packages/babel-traverse/src/scope/index.ts

```diff
@@ -788,6 +814,47 @@ export default class Scope {
     }
   }
 
+  // Must only be called on a scope attached to a program or function path.
+  // Must only be called with functions defined in sloppy mode.
+  // Must not be called with async or generator functions.
+  _registerSloppyFunctionDeclarations(
+    paths: NodePath<t.FunctionDeclaration>[],
+  ) {
+    if (paths.length === 0) return;
+    for (const fnPath of paths) {
+      this._registerSloppyFunctionDeclaration(fnPath);
+    }
+  }
+
+  _registerSloppyFunctionDeclaration(path: NodePath<t.FunctionDeclaration>) {
```

Comment:
Nit: It took me really long do see the difference between this function name and the name of the other function. Could you rename the other one to `_registerMultipleSloppyFunctionDeclarations`? :sweat_smile: